src/Search/Search.Tests/Tests/IndexManagementTests.cs

```diff
@@ -205,9 +205,7 @@ public void DeleteIndexIfNotChangedWorksOnlyOnCurrentResource()
             });
         }
 
-        // TODO,seansaleh: Enable this test once the server side bug is fixed
-        [Fact(Skip = "There is a bug in Azure Search that 404's before checking the ETag. "
-           + "This test exposes this bug and thus is disabled until the server side issue is fixed")]
+        [Fact]
         public void DeleteIndexIfExistsWorksOnlyWhenResourceExists()
```

Comment:
"Works" is meaningless. Can test more precisely describe what is the expected behavior? I.e. Throws 404 or Returns Success